dashboard/v1.1/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "bench-routes-ui",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@types/node": "^12.11.7",
+    "@types/react": "^16.9.9",
+    "@types/react-router-dom": "^5.1.0",
+    "@types/reactstrap": "^8.4.1",
+    "@types/websocket": "^1.0.0",
+    "chart.js": "^2.9.2",
+    "react": "^16.10.2",
+    "react-chartjs-2": "^2.8.0",
+    "react-dom": "^16.10.2",
+    "react-router-dom": "^5.1.2",
+    "react-scripts": "3.2.0",
+    "reactstrap": "^8.4.1",
+    "wait-on": "^3.3.0"
+  },
+  "scripts": {
+    "react-start": "react-scripts start",
+    "react-build": "react-scripts build",
+    "react-test": "react-scripts test",
+    "build": "electron-packager . bench-routes-electron",
+    "eject": "react-scripts eject",
+    "eslint-fix": "eslint . --fix",
+    "tlint-fix": "tslint --fix --project ./tsconfig.json -e src/*.js public/*.js",
+    "prettier-fix": "prettier --write '**/*.tsx'",
+    "lint": "eslint .",
+    "tlint": "tslint src/*.tsx src/**/*.tsx public/*.tsx public/**/*.tsx",
+    "prettier": "prettier '**/*.tsx' --list-different",
+    "prim-tests": "npm run prettier && npm run lint && npm run tlint"
```

Comment:
please make sure that these checks are being run in CI along with the ones in v1. Please update the makefile accordingly. (or .travis.yml).

Reply:
Done.